server-spi-private/src/main/java/org/keycloak/authentication/AbstractAuthenticationFlowContext.java

```diff
@@ -142,6 +142,15 @@
      * @param response Response that will be sent back to HTTP client
      */
     void failure(AuthenticationFlowError error, Response response);
+    
+    /**
+     * Aborts the current flow.
+     *
+     * @param error
+     * @param response Response that will be sent back to HTTP client
+     * @param eventDetails Details about the error event
+     */
+    void failure(AuthenticationFlowError error, Response response, String eventDetails);
```

Comment:
This looks good, but did not you forget to push the change also in the `AuthenticationFlowContext` interface? This PR does not contain it.